lisa/node.py

```diff
@@ -443,6 +443,23 @@ def get_working_path(self) -> PurePath:
 
         return self.get_pure_path(result.stdout)
 
+    def find_partition_with_freespace(self, size_in_gb: int) -> str:
+        if self.os.is_windows:
+            raise NotImplementedError()
```

Comment:
Add a description to make it readable. So the one line message can be helpful in the test results table. It doesn't need to check the log file to see the call trace.